app/registries/menu/loader.rb

```diff
@@ -92,6 +92,12 @@ def self.load
           end
         end
       end
+
+      Manager.map :labs_menu do |menu|
+        menu.sub_menu :lab_features_menu, :caption => N_('Lab Features'), :icon => 'fa fa-flask' do
+          menu.item :dashboard,         :caption => N_('Dashboard')
```

Comment:
@timogoebel is there a better way to do this?  I couldn't get the menu to show up without including at least one `menu.item`.

Reply:
I'm wondering if we should add a default menu item in the first place. If plugins add a "lab feature", they can extend this with their own menu item that actually makes sense.

Reply:
I was thinking that by doing this it makes it obvious which items are in beta (or even alpha) state.  If we just put them into their own menu then we would need some other way of letting users know that these pages may not work 100% and/or may have bugs.

Reply:
@waldenraines: Well, yes. I get the idea. But why should we add "Dashboard" by default? It's not in beta or alpha state.

Reply:
Oh I see, sorry I must have misread the above.

I could not get the menu item to display at all without including this.  I suppose I could remove this and leave the menu item empty and not displaying at all for now until someone adds something to it.  It will just make it more difficult to test.